docs/aca/03-aca-dapr-integration/index.md

```diff
@@ -49,14 +49,26 @@ $UI_APP_PORT=<web frontend ui application https port found under Properties->lau
 
     ```powershell
     cd ~\TasksTracker.ContainerApps\TasksTracker.TasksManager.Backend.Api
-    dapr run --app-id tasksmanager-backend-api --app-port $API_APP_PORT --dapr-http-port 3500 --app-ssl -- dotnet run
+
+    dapr run `
+    --app-id tasksmanager-backend-api `
+    --app-port $API_APP_PORT `
+    --dapr-http-port 3500 `
+    --app-ssl `
+    -- dotnet run
     ```
 
 === ".NET 7 or above"
 
     ```powershell
     cd ~\TasksTracker.ContainerApps\TasksTracker.TasksManager.Backend.Api
-    dapr run --app-id tasksmanager-backend-api --app-port $API_APP_PORT --dapr-http-port 3500 --app-ssl -- dotnet run --launch-profile https
+
+    dapr run `
+    --app-id tasksmanager-backend-api `
+    --app-port $API_APP_PORT `
+    --dapr-http-port 3500 `
+    --app-ssl `
+    -- dotnet run --launch-profile https
```

Comment:
I am leaving `launch-profile` on the same line as it pertains to `dotnet run`, not `dapr run`. Otherwise, it may be very confusing to read.